docs/modules/ROOT/pages/tmail-backend/features/openpaas-integration.adoc

```diff
@@ -0,0 +1,34 @@
+= OpenPaaS Integration
+:navtitle: OpenPaaS Integration
+
+OpenPaaS, developed by Linagora, is a collaborative platform offering features such as calendar management, contact storage, and more. TMail integrates with OpenPaaS to offer enhanced email and collaboration features.
+
+This document describes the key integration points and functionality between TMail and OpenPaaS.
+
+== Features and Integration Points
+=== Integration Overview
+
+TMail interacts with OpenPaaS through two main channels:
+
+- RabbitMQ Messages.
+- REST API Calls to the https://docs.open-paas.org/apis/web/[OpenPaaS API].
```

Comment:
Too technical. That's more for an architecture section.

We should say something like

--------------------

=== Features

TWake mail allows:

 - Ingesting recipients used by local users into their OpenPaaS collected contacts address book
 - Ingesting OpenPaaS contacts into the local auto-complete database
 - Ingesting new events received by email into OpenPaaS calendar
 - Updating attendance in OpenPaaS calendar from the email about the event.
 
In the future it will also display attendance of OpenPaaS calendars in the related emails in Twake Mail.

=== Architecture & implementation details

...